Tools/setup/ubuntu.sh

```diff
@@ -153,7 +144,36 @@ if [[ $INSTALL_NUTTX == "true" ]]; then
 		kconfig-frontends \
 		;
 	fi
-
+	if [[ "${UBUNTU_RELEASE}" == "22.04" ]]; then
+		sudo DEBIAN_FRONTEND=noninteractive apt-get -y --quiet --no-install-recommends install \
+			libunwind-dev \
+			;
+	fi
+	if [[ "${UBUNTU_RELEASE}" == "24.04" ]]; then
+		sudo DEBIAN_FRONTEND=noninteractive apt-get -y --quiet --no-install-recommends install \
+			kconfig-frontends \
+			libncurses6 \
+			libncurses-dev \
+			libncursesw6 \
+			libzmq3-dev \
+			libunwind-dev \
+			;
+
+		# Ubuntu 24.04 requires ZeroMQ
```

Comment:
for what?

Reply:
I have no idea. I couldn't trace it either. Maybe dependency inception? if you have a few minutes it would be ideal to check